test/file_test.py

```diff
@@ -130,6 +130,24 @@ def test_move(self):
         self.assertFalse(os.path.exists(self.path))
         self.assertTrue(os.path.exists(self.copy))
 
+    def test_move_across_filesystems(self):
+        t = LocalTarget(self.path)
+        f = t.open('w')
+        f.write('test_data')
+        f.close()
```

Comment:
Hmm, can't you do the much more elegant context-manager thingie? `with`-statements.